3-networks/README.md

```diff
@@ -40,12 +40,30 @@ The purpose of this step is to :
 1. Merge changes to nonprod with `git checkout -b nonprod` and `git push origin nonprod`
     1. Review the apply output in your cloud build project https://console.cloud.google.com/cloud-build/builds?project=YOUR_CLOUD_BUILD_PROJECT_ID
 
+
 ### Run terraform locally
+1. Change into 3-networks folder.
+1. Run `cp ../build/tf-wrapper.sh .`
+1. Run `chmod 755 ./tf-wrapper.sh`
+1. Rename terraform.example.tfvars to terraform.tfvars and update the file with values from your environment and bootstrap.
+1. Update backend.tf with your bucket from bootstrap. You can run
+```for i in `find -name 'backend.tf'`; do sed -i 's/UPDATE_ME/<YOUR-BUCKET-NAME>/' $i; done```.
+You can run in the 0-bootstap folder `terraform output gcs_bucket_tfstate` to obtain the bucket name.
 
-1. Change into 3-networks folder
-1. Within each envs/ folder rename terraform.example.tfvars to terraform.tfvars and update the file with values from your environment and bootstrap.
-1. Within each envs/ folder, update backend.tf with your bucket name from the bootstrap step.
-1. In this sequence (shared, prod, dev, nonprod), change into each folder within the envs/ folder and run the following commands:
-    1. Run `terraform init`
-    1. Run `terraform plan` and review output
-    1. Run `terraform apply`
+We will now deploy each of our environments(dev/prod/stage) using this script.
+When checked into a VCS, each environment corresponds to a branch and only the corresponding environment is applied.
+
```

Comment:
I think this part cloud be more direct saying something like: 
> when using Cloud Build or Jenkins as your CI/CD tool each environment corresponds to a branch is the repository for `3-networks` step and only the corresponding environment is applied.

binding the branch name to the file system folder is done by the configuration on Cloud Build or Jenkins